src/tests/ftest/daos_perf/daos_perf_large.py

```diff
@@ -27,6 +29,8 @@ def test_large(self):
           Run the combination of above test cases with large number of clients
             on four servers.
 
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,large
```

Comment:
Is this truly a 'hw' test?  The `server_config` section of the yaml file does not require NVMe.

Reply:
Performance should be the answer

Reply:
Ok, then if these tests are being run on HW clusters and performance is of concern, then shouldn't the `daos_perf_large.yaml` file configure the servers that will be started by the test to use the available NVMe devices?  Something like https://github.com/daos-stack/daos/blob/master/src/tests/ftest/io/macsio_test.yaml#L13-L19

Reply:
Yes, you are right yet we need a healthy test to be make sure the yaml changes go in the right direction. The ticket specified in this commit aims to fix daos_perf, we can add the yaml changes there or create a new one, thoughts?

Reply:
Since this PR is changing the tags to run the test in CI and on HW clusters I think it should also set the yaml files correctly - especially in the case of the HW medium tags.  If you want to confirm the changes you could comment out the skip code and the call to `self.run_daos_perf()` to ensure the servers are started correctly with the updated yaml parameters - or just confirm daos_perf still fails after starting the servers with NVMe.

Reply:
Ok but I'll do the test in my local environment, otherwise I'd need a double cycle to confirm a failure and then re-enable the skip for ticket, that is at least 2 days.